docs/api/en/materials/SpriteMaterial.html

```diff
@@ -67,7 +67,7 @@ <h3>[property:Color color]</h3>
 		<p>[page:Color] of the material, by default set to white (0xffffff). The [page:.map] is mutiplied by the color.</p>
 
 		<h3>[property:boolean fog]</h3>
-		<p>Whether or not this material affected by the scene's fog. Default is false</p>
+		<p>Whether or not this material affected by the scene's fog. Default is true</p>
```

Comment:
Instead of changing this section, it's actually better to delete it. Same for the [Chinese version](https://threejs.org/docs/index.html#api/zh/materials/SpriteMaterial).

Properties like `fog` are already documented in the `Material` page. Normally the project only documents such properties again in derived classes if their default value is different.